CONVENTIONS.md

```diff
@@ -290,6 +290,24 @@ Resource request review history:
 * [BZ 1920159](https://bugzilla.redhat.com/show_bug.cgi?id=1920159) --
   for tracking changes in 4.7/4.8 for single-node RAN
 
+##### Allowed use of limits
+
+There is an exception process for adding limits to payload workloads. In the following
+scenarios workloads may be given permission to set limits:
```

Comment:
You could mention that there are exceptions on line 165, referring down to this section.

Gaining that permission is going to be extra work. As a developer, why would I do that? What are the benefits to me, users, the product, etc.?

Reply:
Yeah, would probably add a benefit for each one.  For the current one, it would be:

```
Setting a limit in these cases will ensure that bugs introduced that cause memory leaks will fail in more predictable ways, and more quickly identify the failure to the responsible component.
```

but that's the only advantage (I think)

Reply:
That advantage seems also to come with the downsides mentioned up above. :-) If there's a specific example of a component that would benefit from memory limits, it might help make it easier for the reader to decide whether their thing fits the pattern for even worrying about limits.